### PR TITLE
46elks notifier

### PR DESCRIPTION
**Pull request by jongotlin**

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -55,6 +55,7 @@ with a couple popular SMS services:
 ==============  ====================================  ===========================================================================
 Service         Package                               DSN
 ==============  ====================================  ===========================================================================
+46elks          ``symfony/46elks-notifier``           ``46elks://API_USERNAME:API_PASSWORD@default?from=FROM``
```
Comment (94noni) [inline notifier.rst]: You also need to add a versionadded directive (see same file on v5.4 branch to have hint :))

Comment (OskarStark) [reply notifier.rst]: Versionadded added

**Review by OskarStark — APPROVED** [review]

File: notifier.rst [hunk 2]
```diff
@@ -86,6 +87,10 @@ Vonage          ``symfony/vonage-notifier``           ``vonage://KEY:SECRET@defa
 Yunpian         ``symfony/yunpian-notifier``          ``yunpian://APIKEY@default``
 ==============  ====================================  ===========================================================================
 
+.. versionadded:: 6.1
+
+    The 46elks integration were introduced in Symfony 6.1.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The 46elks integration was introduced in Symfony 6.1.
```

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thanks for your work on this new feature!

**Pull request merged**